src/style.css

```diff
@@ -0,0 +1,89 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars-block {
+  margin-top: 2px;
+}
+
+.stars__row {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  margin-left: 2px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(../src/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
```

Comment:
add styles common for all stars only once to .stars__star, then rewrite background-image for active stars using modifier & nth-child